xamarin/Xamarin.Forms
# [Android] Update the Entry clear button based on FlowDirection

### Description of Change ###

Changed the placement of the entry clear button if the flow direction is right to left

### Issues Resolved ### 

- fixes #8967

### API Changes ###

 
 None

### Platforms Affected ### 


- Android

### Behavioral/Visual Changes ###


If the flow direction is right to left, the clear button will be left-aligned rather than right-aligned.

### Before/After Screenshots ### 


Before:
<img width="363" alt="Screenshot 2020-08-17 at 11 02 21 AM" src="https://user-images.githubusercontent.com/66331238/90417469-5af6a680-e079-11ea-8166-bcf6552a893f.png">

After:
<img width="393" alt="Screenshot 2020-08-17 at 10 59 26 AM" src="https://user-images.githubusercontent.com/66331238/90417444-53cf9880-e079-11ea-884f-41b2103771bd.png">

### Testing Procedure ###


Create an entry with a clear button, set the flow direction to right-to-left, and make sure the clear button is left-aligned.

### PR Checklist ###
